SConstruct

```diff
@@ -349,7 +349,10 @@ if selected_platform in platform_list:
         else: # always enable those errors
             env.Append(CCFLAGS=['-Werror=return-type'])
 
-    suffix = "." + selected_platform
+    if (hasattr(detect, 'get_program_suffix')):
```

Comment:
Since it will only be used for the server platform, and will likely be obsoleted in 3.2 when we merge it into the other platforms as a build option/CLI switch, it might be better to just hack this in `platform/server/SCsub` since there's already a platform check their for the source files to include.

Reply:
Ah, right. Guess that's fine this way then, to keep compatibility for people with their own build scripts for the "X11" server.